testData/folding/switchSelectCaseStatement.go

```diff
@@ -0,0 +1,41 @@
+package foo
+
+func main() <fold text='{...}'>{
+	for <fold text='{...}'>{
+		println()
+	}</fold>
+	
+	sum := 0
+	for i := 0; i < 10; i++ <fold text='{...}'>{
+		sum += i
+	}</fold>
+
+	ch := make(chan int)
+	select <fold text='{...}'>{
+	case _ <- ch:
+		println("demo")
+	case _ <- ch:<fold text='{...}'>
+		println("demo")
+		println("demo")</fold>
+	}</fold>
+
+	switch true <fold text='{...}'>{
+	case 1 == 2:
+		println("demo")
+
+	case 2 == 3:<fold text='{...}'>
+		println("demo")
+		println("demo")</fold>
+
+	default:<fold text='{...}'>
+		println("demo")
+		println("demo")
+		println("demo")</fold>
+	}</fold>
+
+	switch true {
```

Comment:
It's because of the clause here: https://github.com/go-lang-plugin-org/go-lang-idea-plugin/blob/16bfaadc38887de75d7d6c102271abef1b993470/src/com/goide/editor/GoFoldingBuilder.java#L45 I could extrat the size limit as a parameter if you think it makes more sense.